packages/contracts/contracts/LPRewards/SushiSwapOhmLqtyRewarder.sol

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.6.11;
+
+import "./Interfaces/IRewarder.sol";
+import "./Dependencies/BoringERC20.sol";
+import "./Dependencies/BoringMath.sol";
+
+
+// Based on:
+// https://github.com/sushiswap/sushiswap/blob/master/contracts/mocks/RewarderMock.sol
+contract SushiSwapOhmLqtyRewarder is IRewarder {
+    using BoringMath for uint256;
+    using BoringERC20 for IERC20;
+
+    uint256 private immutable ohmRewardMultiplier;
+    IERC20 private immutable ohmToken;
+    uint256 private immutable lqtyRewardMultiplier;
+    IERC20 private immutable lqtyToken;
+    uint256 private constant REWARD_TOKEN_DIVISOR = 1e18;
+    address private immutable MASTERCHEF_V2;
+
+    constructor (
+        uint256 _ohmRewardMultiplier,
+        IERC20 _ohmToken,
+        uint256 _lqtyRewardMultiplier,
+        IERC20 _lqtyToken,
+        address _MASTERCHEF_V2
+    ) public {
+        ohmRewardMultiplier = _ohmRewardMultiplier;
+        ohmToken = _ohmToken;
+        lqtyRewardMultiplier = _lqtyRewardMultiplier;
+        lqtyToken = _lqtyToken;
+        MASTERCHEF_V2 = _MASTERCHEF_V2;
+    }
+
+    function onSushiReward (uint256, address, address to, uint256 sushiAmount, uint256) onlyMCV2 override external {
+        // OHM rewards
+        uint256 ohmPendingReward = sushiAmount.mul(ohmRewardMultiplier) / REWARD_TOKEN_DIVISOR;
```

Comment:
I noticed the OHM token uses 9-digit decimal precision:
https://etherscan.io/address/0x383518188c0c6d7730d91b2c03a03c837814a899#readContract
 and SUSHI and LQTY use 18.

It seems here that  `ohmRewardMultiplier` and `lqtyRewardMultiplier` variables are 18-digit decimal precision, and that's why they're divided by `REWARD_TOKEN_DIVISOR`.

Perhaps we can add comments to note that:
- These multiplier variables are 18-digit precision
- The value of `ohmRewardMultiplier` should take into account the different decimal precision of SUSHI and OHM when it is set at construction.

e.g. if we wanted to reward 2 OHM per SUSHI, the raw value of `ohmRewardMultiplier` should be `2e9`.  Then when calculating the OHM pending reward, we'd get: 

```
ohmPendingReward = sushiAmount * 2e9 / 1e18

= sushiAmount * 2 / 1e9
```

Which is equivalent to 2 OHM per SUSHI, when converted into OHM 9-digit precision.

Basically at deployment we just need to remember to pass  an `_ohmRewardMultiplier` on the order of `1e9`, and `_lqtyRewardMultiplier` should be on the order of `1e18`.

I think tests don't catch this because the OZ ERC20.sol hardcodes `decimals = 18`.